packages/openapi-v3/src/__tests__/unit/filter-schema.unit.ts

```diff
@@ -22,6 +22,7 @@ describe('filterSchema', () => {
     expect(MyUserModel.definition.name).to.eql('my-user-model');
     expect(schema).to.eql({
       title: 'my-user-model.Filter',
+      type: 'object',
```

Comment:
IIUC, `openapi-v3` package uses AJV only for tests (via `devDependencies`), therefore the AJV upgrade should not affect our users, it should not be breaking any existing code. If my understanding is correct, then you need to move changes in `packages/openapi-v3` to a different commit, one without `BREAKING CHANGE` section.

Reply:
Actually, I see that you have to modify production code to add `type: 'object'` to places where it was not added before. Maybe that's just a regular feature? If it's a breaking change, then IMO we should describe it from the perspective of what `openapi-v3` consumers are going to observe - i.e. we are adding `type: 'object'` now in order to support AJV v7. 

Thoughts?

Reply:
AJV v7 is stricter - by default requiring `type: 'object'`.